crates/parser/tests/snapshots/statements/valid/0047@4.snap

```diff
@@ -0,0 +1,47 @@
+---
+source: crates/parser/tests/statement_parser_test.rs
+description: "\nCREATE DATABASE x LOCATION DEFAULT;"
+---
+Parse {
+    cst: SourceFile@0..36
+      Newline@0..1 "\n"
+      CreatedbStmt@1..36
+        Create@1..7 "CREATE"
+        Whitespace@7..8 " "
+        Database@8..16 "DATABASE"
+        Whitespace@16..17 " "
+        Ident@17..18 "x"
+        Whitespace@18..19 " "
+        DefElem@19..27
+          Location@19..27 "LOCATION"
+        Whitespace@27..28 " "
+        Default@28..35 "DEFAULT"
```

Comment:
Should also be part of DefElement

Reply:
fixed in 45f88be683ba728c8454d02c6a2feb1349f3ce58